pom.xml

```diff
@@ -61,6 +61,41 @@
         <version>1.22</version>
         <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-step-api</artifactId>
+      <version>2.3</version>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-support</artifactId>
+      <version>2.6</version>
```

Comment:
I think you added it for the `workflow-step-api`, and not the `workflow-support` dependency.